rebasehelper/spec_hooks/pypi_url_fix.py

```diff
@@ -62,6 +63,6 @@ def _transform_url(cls, line):
         Perform predefined URL transformations
         """
         for trans in cls.SOURCES_URL_TRANSFORMATIONS:
-            if re.search(r".*https?://pypi.python.org/.*", line):
+            if re.search(r".*" + trans[0] + r".*", line):
```

Comment:
First, why prepend and append `r'.*'`? `re.search()` would find it anyway.
Second, why two regexps, when one is a subset of the other?
Third, this could be done with just `re.sub()` or `re.search()`, using both is unnecessary.